.github/workflows/release.yml

```diff
@@ -66,11 +66,11 @@ jobs:
 
       # release info is now in main so we can continue as before
       - name: create release with last commit
-        uses: actions/create-release@v1
         if: steps.changelog.outputs.skipped == 'false'
+        uses: ncipollo/release-action@v1
```

Comment:
Is there a reason that you selected this action?

Other than that, looks good.

Reply:
Great question, Dan. 

Yes, there is a specific reason for that. I tested out every maintained action mentioned in the https://github.com/actions/create-release README, and only this one and the one mentioned above have support for writing the body of the release. I chose this particular action because it offers better customization options.